ui-react/packages/atlasmap/src/Views/NamespaceTableView.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { FunctionComponent, ReactElement } from "react";
+
+import {
+  Bullseye,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  Title,
+  Level,
+  LevelItem,
+  Tooltip,
+  Button,
+} from "@patternfly/react-core";
+import { TableIcon, PlusIcon } from "@patternfly/react-icons";
+import { css, StyleSheet } from "@patternfly/react-styles";
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  IRowData,
+  textCenter,
+  ICell,
+} from "@patternfly/react-table";
+
+import { MainContent } from "../Layout";
+import { IAtlasmapDocument } from ".";
+import { Actions } from "../UI";
+import { ISourceColumnCallbacks } from "./ColumnMapperView";
+
+const styles = StyleSheet.create({
+  title: { marginTop: "1rem", marginBottom: "1rem" },
+});
+
+export interface INamespaceTableProps {
+  sources: IAtlasmapDocument[];
+  sourceEvents: ISourceColumnCallbacks;
```

Comment:
I’m not a fan of this. Better have a copy of the interface than coupling the two views together since they are not related